torch/onnx/symbolic_opset10.py

```diff
@@ -958,6 +958,7 @@ def quantized_conv_transpose1d(
     bias,
     stride,
     padding,
+    output_padding,
```

Comment:
Note sure how to address this linter error. These are not user-facing functions and not feeding in `output_padding` was a bug to begin with.

Reply:
I was confused for a moment how this syntactically worked, as I was assuming the C++ `quantized::conv_transpose1d` signature did not change. But later realized the assumption is not true.